vendor/brave-ios/BUILD.gn

```diff
@@ -78,6 +78,7 @@ ios_framework_bundle("brave_rewards_ios_framework") {
     "Shared/BATCommonOperations.mm",
     "Shared/NSURL+Extensions.mm",
     "Shared/RewardsLogStream.mm",
+    "Shared/Logger.mm",
```

Comment:
Also include headers here, as seen in `master` branch: https://github.com/brave/brave-core/blob/9ba2175e86c1a58311a11814dee06b5579f7f549/vendor/brave-ios/BUILD.gn#L65-L80